src/test/java/org/folio/rest/api/IsbnNormalizationTest.java

```diff
@@ -0,0 +1,194 @@
+package org.folio.rest.api;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import static org.folio.rest.api.StorageTestSuite.storageUrl;
+import static org.folio.util.StringUtil.urlEncode;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.rest.support.ApiTests;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.builders.RequestItemSummary;
+import org.folio.rest.support.builders.RequestRequestBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
+import lombok.SneakyThrows;
+
+public class IsbnNormalizationTest extends ApiTests {
+  private static final String REQUEST_STORAGE_URL = "/request-storage/requests";
+  private final UUID isbnIdentifierId = UUID.fromString("8261054f-be78-422d-bd51-4ed9f33c3422");
+
+  @SneakyThrows
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(storageUrl(REQUEST_STORAGE_URL));
+  }
+
+  @SneakyThrows
+  @Test
+  public void canSearchForFirstIsbnWithAdditionalHyphens() {
+    createRequests("Interesting Times", "Uprooted");
+
+    find("isbn = 0-552-16754-1",      "Interesting Times");
+  }
+
+  @SneakyThrows
+  @Test
+  public void canSearchForFirstIsbnWithAdditionalHyphenAndTruncation() {
+    createRequests("Interesting Times", "Uprooted");
+
+    find("isbn = 05-5*",              "Interesting Times");
```

Comment:
What is the intent behind the horizontal alignment of the second parameter across tests?

Reply:
The way I originally wrote this, the tests were all the same line length, and close together, so that the titles and searches all lined up across tests.  It's been fiddled with so much that formatting no longer works.  I'll remove in the next commit.